src/workflow/ParameterManager.py

```
@@ -16,14 +17,36 @@ class ParameterManager:
+
+        Args:
+            tool: Name of the TOPP tool (e.g., "CometAdapter")
+
+        Returns:
+            True if ini file exists (created or already existed), False if creation failed
+        """
+        ini_path = Path(self.ini_dir, tool + ".ini")
+        if ini_path.exists():
+            return True
+        try:
+            subprocess.call([tool, "-write_ini", str(ini_path)])
```

Ruff (0.15.4)

[error] `subprocess` call: check for execution of untrusted input

(S603)